src/components/dashboard/DashboardPage.jsx

```diff
@@ -79,6 +80,7 @@ const DashboardPage = () => {
           description="Alert message shown to a learner on enterprise dashboard courses tab."
         />
       </Alert>
+      <BrowseAndRequestAlert />
```

Comment:
[question] wouldn't it be better to handle the conditional rendering here instead of the inside the component? i.e., 
```javascript
 !data?.hasBnrEnabledPolicy && <BrowseAndRequestAlert />

Reply:
The early return pattern used is preferred on this approach because it clearly separates rendering eligibility logic from the actual UI code and we don't have to call  `useBFF` in the parent.